loadtest/README.md

```diff
@@ -0,0 +1,111 @@
+# Load testing
+
+This directory contains [locust](https://locust.io) load testing tasks to load test the Global.health APIs.
+
+Set-up and initial load testing was tracked in [this github issue](https://github.com/globaldothealth/list/issues/1263).
+
+## TL;DR results
+
+Dev starts to cpu-choke at ~10 QPS (simulated with ~20 users and a spawn rate of 1), it's using the allowed 250m vCPUs fully while the RAM usage doesn't increase at all (50Mb->60Mb).
```

Comment:
Does cpu choke basically mean we move from "ballpark reasonable" latency to "totally unreasonable" latency at 10QPS? Since we don't have latency SLOs around which to frame stress test results, I think this description works if that's what you mean.

Reply:
yes I rephrased, that's what I meant, latency was getting unreasonable when CPU is over the limits.